code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -45,7 +45,7 @@ Credit where due:
 ///////////
 
 /proc/is_servant_of_ratvar(mob/M)
-	return istype(M) && M.mind && M.mind.has_antag_datum(/datum/antagonist/clockcult)
+	return istype(M) && M.mind && M.mind.has_antag_datum(/datum/antagonist/clockcult) && !isobserver(M)
```

Comment:
you could probably just change 47 to be `(mob/living/M)` to get the desired effect without doing a check to see if it's a mob AND a check to see if it isn't a dead mob.

Reply:
done